### ISSUE TITLE
Release ID not being found when created in previous workflow job 

### ISSUE THREAD

**Issue by 09wattry** [skx/github-action-publish-binaries#20]
This is a feature request to add an optional release id input so that releases created in the workflow template can be used. This will allow us to automate the whole release cycle. Since we only add a branch into master when we are ready to make a release.

We are using a GitHub action workflow to create a release, create binaries, and finally publish them using github-action-publish-binaries to the release that was created in the first job.

## How we'd like to run our workflow.

```
name: Create Releases

on:
  push:
    branches: [ master ]

jobs:
  create-release:
    runs-on: ubuntu-latest
    outputs:
      releaseId: ${{ steps.create-release.output.id }}
    steps:
      - name: Create Release
        id: create_release
        uses: actions/create-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          tag_name: 1.0.0
          release_name: 1.0.0

  build-binaries:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: 12
      - run: npm install
      - run: npm run build
      - uses: actions/upload-artifact@v1
        with:
          name: dist
          path: dist

  publish-binaries:
    runs-on: ubuntu-latest
    needs: [create-release, build-binaries]
    steps:
      - uses: skx/github-action-publish-binaries@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          releaseId: ${{ needs.create-release.outputs.releaseId }}
          args: 'dist/*'

```

My suggestion is to use the release Id if one is provided or just use the event if there is not one. This would allow us to make manual releases on feature branches and still automate releases to master.

I'm not a super proficient bash scripter so I will put in a PR, feel free to leave me some advice.

Thanks in advance.

**Comment by skx** [comment]
Looks like this will be fixed in #21 - thanks for the contribution, just offered minor feedback there.

**Comment by skx** [comment]
Closed in #21.
